Sources/PostgresNIO/Connection/PostgresConnection+Database.swift

```diff
@@ -3,7 +3,7 @@ import Logging
 extension PostgresConnection: PostgresDatabase {
     public func send(
         _ request: PostgresRequest,
-        logger: Logger
+        logger: Logger?
```

Comment:
Same question here re: `= nil`.

Reply:
Yeah, for this method it probably should default to `nil` if unspecified.